amplipi/streams.py

```diff
@@ -1335,16 +1336,16 @@ def _activate(self, vsrc: int):
 
       self.proc = subprocess.Popen(args=lms_args)
     except Exception as exc:
-      print(f'error starting lms: {exc}')
+      logging.exception(f'error starting lms: {exc}')
 
   def _deactivate(self):
     if self._is_running():
       try:
         self.proc.terminate()
         self.proc.communicate()
       except Exception as e:
-        print(f"failed to terminate LMS stream {self.name}: {e}")
-        print("forcefully killing.")
+        logging.exception(f"failed to terminate LMS stream {self.name}: {e}")
+        logging.info("forcefully killing.")
```

Comment:
these deserve to be the same log level. imagine setting log level to "INFO"; you'd only get an obtuse line saying "forcefully killing", without it specifying what it's killing or why.

Reply:
I made it a single logging instance in the recent commit
`logging.exception(f"failed to terminate LMS stream {self.name}: {e} \nforcefully killing.)`
This seems like a good practice to me to retain the newline but without the risk of other prints / logging events happening between the two instances, \n for newline seems uncommon so I wanted to call attention to it in case that's intentionally avoided elsewhere

Reply:
It'll look a little funny in the logs; normally we don't want to be concerned with formatting/rendering log lines and just emitting them, but it'll do.